misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt

```diff
@@ -100,6 +104,52 @@ abstract class PathFeatureAssertionsSpec(
         }
     }
 
+    describeFun("val ${fileNameFeature.name}") {
+        val fileNameVal = fileNameFeature.lambda
+
+        context("File with extension") {
+            it("toBe(my.txt) holds") {
+                expect(Paths.get("a/my.txt")).fileNameVal().toBe("my.txt")
+            }
+            it("toBe(my.txt) fails") {
```

Comment:
This test case does not make sense in this context `"File with extension"` -> your case does not have an extension.
However, I suggest you rename the context to `context("path a/my.txt")` as fileName does not have anything to do with *does it have an extension or not?*